expenses/models.py

```diff
@@ -1,7 +1,12 @@
 from django.db import models
+from django.utils import timezone
 
 
 class Expenses(models.Model):
+    # house_id = models.ForeignKey('user_id', on_delete=models.CASCADE)
```

Comment:
What about this comment?

Reply:
> What about this comment?

I'll be glad if you can help me with the connection between expense and house (expense needs the house_id of it's house)
thanks! :) 
@adiabramovitch

Reply:
Are we going to remove this comment when the House model will be merged to master?
I think it is there already: https://github.com/redhat-beyond/Xpense/blob/main/house/models.py
and you can try to remove it and test the interaction between house and expenses

Reply:
I fixed it